src/App.js

```diff
@@ -1,14 +1,89 @@
-import React from 'react';
+import React from "react";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import TodoList from "./components/TodoComponents/TodoList";
+import SimpleStorage from "react-simple-storage";
+
+import "./minireset.css";
+import "./App.css";
+
+const todos = [
+  {
+    task: "Organize Garage",
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: "Bake Cookies",
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todos: todos,
```

Comment:
there's a couple different ways to do this to avoid having a double-name convention happening here (even though it does work), that's to either change one of the state property `todos` or the initially-defined array `todos` to a different name, or to define the initial array _inside_ the state object instead of outside of it: ```this.state ={todos: [
  {
    task: "Organize Garage",
    id: 1528817077286,
    completed: false
  },
  {
    task: "Bake Cookies",
    id: 1528817084358,
    completed: false
  }
]}